ush/build.sh

```diff
@@ -30,8 +30,6 @@ set -x
 
 # Set CONTROLPATH variable to user develop installation
 CONTROLPATH="$DIR_ROOT/../develop/install/bin"
-CMAKELIBS=".so .a"
-CMAKE_OPTS+=" -DCMAKE_FIND_LIBRARY_SUFFIXES=${CMAKELIBS}"
```

Comment:
I believe these should stay.

Reply:
These options were added at the initial tests for porting to Gaea; not needed. GSI builds and run successfully without setting the lib. suffixes explicitly